src/main/java/Duke.java

```diff
@@ -1,10 +1,49 @@
+import java.util.Objects;
+import java.util.Scanner;
+import static java.lang.Integer.parseInt;
+
 public class Duke {
+    private static Task[] tasks = new Task[100];
+    private static int taskCount = 0;
+
+    public static void addTask(Task task) {
+        tasks[taskCount] = task;
+        taskCount += 1;
+        DukePrinter.printMessage("added: " + task.getDescription());
+    }
+    public static void markTask(int taskIndex) {
+        //TODO: ensure taskIndex is not <= 0
+        tasks[taskIndex - 1].setDone(true);
+        DukePrinter.printMessage("Nice! I've marked this task as done:\n" + "    " + tasks[taskIndex - 1].getTaskString());
+    }
+    public static void unmarkTask(int taskIndex) {
+        //TODO: ensure taskIndex is not <= 0
+        tasks[taskIndex - 1].setDone(false);
+        DukePrinter.printMessage("OK, I've marked this task as not done yet:\n" + "    " + tasks[taskIndex - 1].getTaskString());
```

Comment:
Perhaps you may want to split the line length to ensure it fits the required number of chars per line